report-generator/.cqfd/docker/Dockerfile

```diff
@@ -10,10 +10,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	asciidoc \
 	bash \
 	locales \
+	python3-pip \
 	ruby \
 	ruby-asciidoctor-pdf \
 	xmlstarlet \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN sed -i "s/# en_US\.UTF-8 UTF-8/en_US\.UTF-8 UTF-8/" /etc/locale.gen
 RUN locale-gen
+
+RUN pip3 install matplotlib
```

Comment:
Should be moved in the apt RUN statement if conserved to prevent creating temporary docker images (docker creates snapshots for each statement).